frontend/src/pages/ExperimentDetails.tsx

```diff
@@ -109,7 +109,7 @@ interface ExperimentDetailsState {
 
 class ExperimentDetails extends Page<{}, ExperimentDetailsState> {
 
-  private _runlistRef = React.createRef<RunList>();
+  protected _runlistRef = React.createRef<RunList>();
```

Comment:
Doesn't seem necessary?

Reply:
Reverted. At some point was working on a test involving using this explicitly.